src/test/pretty/issue-80832.rs

```diff
@@ -0,0 +1,5 @@
+// pretty-mode:expanded
```

Comment:
Could you rename the test to something more descriptive? e.g. `pretty-expanded-with-remapped-path-issue-80832.rs`, there's interest in avoiding new tests just named `issue-XXXXX.rs`.

Reply:
Sure, though I'd got comments going the other way in previous PRs.

Reply:
OK done